src/opencv.mk

```diff
@@ -39,10 +40,12 @@ define $(PKG)_BUILD
       -DBUILD_TIFF=OFF \
       -DBUILD_JASPER=OFF \
       -DBUILD_JPEG=OFF \
+      -DBUILD_WEBP=OFF \
+      -DBUILD_PROTOBUF=ON \
```

Comment:
Is it using MXE's protobuf or its own copy? If former, add protobuf to dependencies, please. (It seems to be an indirect dependency, but if a library is used directly, it should be a direct dependency.)

Reply:
I explained it above. Those flags are for the bundled versions. I added that protobuf because MXE's protobuf breaks it.

Reply:
Can you add a comment that MXE's protobuf breaks the build and that we use the bundled protobuf, please?

Reply:
Done. Also took the liberty of squashing all commits.